packages/libluajit/build.sh

```diff
@@ -3,8 +3,10 @@ TERMUX_PKG_DESCRIPTION="Just-In-Time Compiler for Lua"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=1:2.1.0-beta3
-TERMUX_PKG_SRCURL=https://github.com/LuaJIT/LuaJIT/archive/v${TERMUX_PKG_VERSION:2}.tar.gz
-TERMUX_PKG_SHA256=409f7fe570d3c16558e594421c47bdd130238323c9d6fd6c83dedd2aaeb082a8
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_COMMIT="787736990ac3b7d5ceaba2697c7d0f58f77bb782"
```

Comment:
Please use just `_COMMIT`, that's how [other packages do it](https://github.com/termux/termux-packages/search?q=_commit)

Reply:
Sorry . I've fixed it now